source/funkin/util/ThreadUtil.hx

```diff
@@ -16,6 +16,20 @@ class ThreadUtil
   public static var loopingThreads:Map<String, ThreadParams> = [];
   #end
 
+  /**
+   * Creates a Thread that executes `job` indefinitely.
```

Comment:
This comment is a bit misleading since `Thread.create()` only executes the specific function once:

```haxe
	/**
		Creates a new thread that will execute the `job` function, then exit.

		This function does not setup an event loop for a new thread.
	**/
	public static function create(job:()->Void):Thread;
```